build-tools/xaprepare/xaprepare/Steps/Step_Get_Windows_Binutils.cs

```diff
@@ -301,7 +303,7 @@ void Extract (Stream src, Stream dest, uint crc32FromHeader)
 		{
 			uint fileCRC = 0;
 			int fread = 0;
-			var crc32 = new CRC32 ();
+			var crc32 = new SafeProxy ();
```

Comment:
The `SafeProxy` class name  is rather meaningless in this context. I understand that the point here is to use 3rd party code without mingling it with ours, but can we perhaps introduce a dummy `class CRC32 : SafeProxy` to make this line less cryptic?

Reply:
Done.